test/docker/wait-for-cassandra.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
This script is necessary because on Travis CI we start `docker-compose` as detached (for obvious reasons), but we want to wait for C* to be ready before running the tests. In some way, this is similar to what we had before in the Travis config file:

```
for try in {1..12}; do (nodetool version > /dev/null 2>&1 && break) || sleep 5; done
```